demo/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -128,6 +140,13 @@ const MeetingForm: React.FC = () => {
         value={region}
         label="Select AWS region"
       />
+      <FormField
+        field={Select}
+        options={getFormattedOptionsForSelect(LOG_LEVELS)}
+        onChange={handleLogLevelChange}
+        value={logLevel}
+        label="Select SDK log level"
+      />
```

Comment:
I don't think this should be a part of the form. It's really just for developers, not for users

Reply:
Hmm, Ok, I thought it would be good to do that selection from UI itself without needing to change the code elsewhere. But yes, we can go the prop route as you said above.

Reply:
If we want to be able to toggle the log level in our demo, we could probably just stick it in the query params

?logging=info or something

Reply:
Thats cool, dint thought of this approach. Thanks will add it.